src/averageAges.js

```diff
@@ -37,7 +48,18 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => person.sex === 'f');
+  let womenWithChildren;
+
+  typeof withChildren !== 'undefined'
```

Comment:
Use ternary operator instead of a separate assignment for `womenWithChildren`. 🌟